src/main/java/nl/hsac/fitnesse/fixture/Environment.java

```diff
@@ -6,28 +6,7 @@
 import freemarker.template.DefaultObjectWrapperBuilder;
 import freemarker.template.Template;
 import nl.hsac.fitnesse.fixture.slim.HttpClientSetup;
-import nl.hsac.fitnesse.fixture.util.DatesHelper;
-import nl.hsac.fitnesse.fixture.util.Formatter;
-import nl.hsac.fitnesse.fixture.util.FreeMarkerHelper;
-import nl.hsac.fitnesse.fixture.util.HtmlCleaner;
-import nl.hsac.fitnesse.fixture.util.HttpClient;
-import nl.hsac.fitnesse.fixture.util.HttpResponse;
-import nl.hsac.fitnesse.fixture.util.JsonHelper;
-import nl.hsac.fitnesse.fixture.util.JsonPathHelper;
-import nl.hsac.fitnesse.fixture.util.LineEndingHelper;
-import nl.hsac.fitnesse.fixture.util.MapHelper;
-import nl.hsac.fitnesse.fixture.util.NamespaceContextImpl;
-import nl.hsac.fitnesse.fixture.util.ProgramHelper;
-import nl.hsac.fitnesse.fixture.util.ProgramResponse;
-import nl.hsac.fitnesse.fixture.util.PropertiesHelper;
-import nl.hsac.fitnesse.fixture.util.ReflectionHelper;
-import nl.hsac.fitnesse.fixture.util.SecretMasker;
-import nl.hsac.fitnesse.fixture.util.TextFormatter;
-import nl.hsac.fitnesse.fixture.util.TimeoutHelper;
-import nl.hsac.fitnesse.fixture.util.XMLFormatter;
-import nl.hsac.fitnesse.fixture.util.XPathHelper;
-import nl.hsac.fitnesse.fixture.util.XmlHttpResponse;
-import nl.hsac.fitnesse.fixture.util.ZipHelper;
+import nl.hsac.fitnesse.fixture.util.*;
```

Comment:
Please leave the individual imports (I suggest you configure your editor to only use `*` imports if more than 999 imports are done from one package (i.e. never).

Star-imports can give nasty side effects if the packages you are using get more (or less) classes